embedded-keycloak/src/test/resources/application-disabled.yml

```diff
@@ -0,0 +1,3 @@
+embedded:
```

Comment:
you need to specify such properties only in bootstrap-* files. you don't need them to be present both in application and bootstrap files